src/libScilla/ScillaIPCServer.cpp

```diff
@@ -230,6 +241,63 @@ bool ScillaIPCServer::fetchExternalStateValue(const std::string &addr,
   return true;
 }
 
+void ScillaIPCServer::fetchStateJsonI(const Json::Value &request,
+                                      Json::Value &response) {
+  INC_CALLS(GetCallsCounter());
+  const auto address = Address{request["addr"].asString()};
+  const auto vname = request["vname"].asString();
+  if (!request["indices"].isArray()) {
+    return;
```

Comment:
Really? Do you not want to return a Result<> here and actually provide an error?